subprojects/core/src/main/java/org/gradle/api/internal/tasks/DefaultTaskInputFilePropertySpec.java

```diff
@@ -110,13 +110,13 @@ public TaskInputFilePropertyBuilderInternal withPathNormalizationStrategy(PathNo
     }
 
     @Override
-    public TaskInputFilePropertyBuilderInternal withSnapshotter(Class<? extends FileCollectionSnapshotter> snapshotter) {
+    public TaskInputFilePropertyBuilderInternal withNormalizer(Class<? extends PropertyNormalizer> snapshotter) {
         this.snapshotter = snapshotter;
         return this;
     }
 
     @Override
-    public Class<? extends FileCollectionSnapshotter> getSnapshotter() {
+    public Class<? extends PropertyNormalizer> getSnapshotter() {
```

Comment:
If we can, we should change the method to `getNormalizer`.

Reply:
Done.